test/lib/dynamo/encoder_test.exs

```diff
@@ -86,4 +86,16 @@ defmodule ExAws.Dynamo.EncoderTest do
     assert Encoder.encode(nil) == %{"NULL" => true}
     assert Encoder.encode(%{"key" => nil}) == %{"M" => %{"key" => %{"NULL" => true}}}
   end
+
+  test "encoder skips struct fields that are marked as excepted" do
+    user_except =
+      %Test.ExceptUser{email: "foo@bar.com", name: "Bob", age: 23, secret: "secret information"}
+      |> Encoder.encode_root()
+
+    assert %{
+             "age" => %{"M" => %{"N" => %{"S" => "23"}}},
+             "email" => %{"M" => %{"S" => %{"S" => "foo@bar.com"}}},
+             "name" => %{"M" => %{"S" => %{"S" => "Bob"}}}
+           } = Encoder.encode_root(user_except)
```

Comment:
Thanks for your work on this, @wjwitek - I'm just taking a passing glance here, I haven't read the code in this repo in ages...

Would this `assert` be better served with a `==`? If I'm not mistaken, the single `=` would still match if the righthand side had a `"secret"` key, so maybe the `==` would be a stronger assertion that indeed, `"secret"` has been dropped?

Reply:
Yes, you're right, thanks for catching that. I've changed the assert.

Reply:
Thank you! Glad that was helpful.

Reply:
Thanks for catching that, @darrenklein - I would have totally missed it.

Reply:
They call me Eagle Eye... ok, no one calls me that, but I wish I had a cool nickname.